index.html

```diff
@@ -0,0 +1,96 @@
+			</div>
+		</header>
+		<div class="main-body">
+			<img class="profile-img" src="/images/head-shot.jpg" alt="profile head shot of Anessa Ortner">
+			<article class="bio">
+				<section>
+					<h3>Who am I?</h3>
+					<p>I'm a high energy curious person in this world.  I have personal interests that range from nature to the depths of an engine block.  I grew up in a varied environment of artist lectures and physics textbooks dotting the family library.  If I can get into it I will.  My goals are to explore everything from knowledge to the four corners of the world.  As a visual person I want to see, create, and design the world around me.  As a hyperactive imaginative I will never stop asking "why?" and strive for my next challenge.</p>
+				</section>
+				<section>
+					<h3>What do I do?</h3>
+					<p>As a professional designer and developer I strive to achieve the biggest viewer impact with minimal forms.  I've worked in industries ranging from print to digital distribution, direct and indirect customer and client production, and had the amazing opportunity to consult and design for varied corporate clients.  I want to work FOR you and WITH you to achieve YOUR goals, no matter the size.</p>
+				</section>
+			</article>
+			<div class="professional-info">
+				<article class="upcoming">
+					<h3>Upcoming Projects:</h3>
+					<section>
+						<div class="project-preview">
+							<a href="http://www.swankypress.com/">SWANKY PRESS WEBSITE REFRESH</a>
+						</div>
+					</section>
+				</article>
+				<article class="clients">
+					<h3>Clients:</h3>
+					<a href="http://www.swankypress.com">
+						<img src="/images/swanky-logo.png" alt="swanky press logo">
+					</a>
+					<a href="http://www.donnaortner.com">
+						<img src="/images/donna-o-logo.png" alt="Donna Ortner Yoga logo">
+					</a>
+					<a href="http://www.pilatesplacenashville.com">
+						<img src="/images/pilates-place-logo.png" alt="Pilates Place logo">
+					</a>
+				</article>
+			</div>
+			<article class="special-links">
+				<section class="resrouces">
+					<h3>Informative Resources:</h3>
+					<ul>
+						<li>
+							<a href="https://developer.mozilla.org/en-US/"><span>Mozilla Developer Network</span></a>
+						</li>
+						<li>
+							<a href="https://github.com/"><span>Github</span></a>
+						</li>
+						<li>
+							<a href="http://www.adobe.com/creativecloud.html"><span>Adobe CC</span></a>
+						</li>
+					</ul>
+				</section>
+				<section class="inspiration">
+					<h3>Inspirational Resources:</h3>
+					<ul>
+						<li>
+							<a href="http://www.spectrumfantasticart.com/"><span>Spectrum Fantastic Art</span></a>
+						</li>
+						<li>
+							<a href="http://www.spacex.com/"><span>SpaceX</span></a>
+						</li>
+					</ul>
+				</section>
+			</article>
+		</div>
+		<footer class="main-footer">
+			<section class="contact">
+				<h5>Contact:</h5>
+				<a href= "mailto:shadowglacierdesigns@gmail.com">Send Email</a>
```

Comment:
this looks kinda awkward.  How about a letter icon that when you click on it does the mailto?  That would better mirror the icons you have in the  connect section